torch/utils/data/dataloader.py

```diff
@@ -341,7 +366,7 @@ def __init__(self, loader):
         self._base_seed = torch.empty((), dtype=torch.int64).random_(generator=loader.generator).item()
         self._num_yielded = 0
 
-    def __iter__(self):
+    def __iter__(self) -> '_BaseDataLoaderIter':
```

Comment:
This one doesn't need to be quoted, right?

Reply:
I'm getting `NameError: name '_BaseDataLoaderIter' is not defined` if I remove the quotes (mypy 0.770)

Reply:
Sad, mypy limitations :/